webrtc.html

```diff
@@ -743,6 +743,11 @@
             <p>Let <var>p</var> be a new promise.</p>
           </li>
           <li>
+            <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+            <code>true</code>, reject <var>p</var> with
+            <code>InvalidStateError</code> and abort these steps.</p>
+          </li>
+          <li>
```

Comment:
We don't need to check the closed state here since the 'enqueue' algorithm does the check before executing the operation.